src/docs/clients/cocoa/migration.mdx

```diff
@@ -13,23 +13,23 @@ tags: []
 
 Update your `Podfile` to include `:subspecs => ['Core', 'KSCrash']`:
 
-  ```ruby
-  source 'https://github.com/CocoaPods/Specs.git'
-  platform :ios, '8.0'
-  use_frameworks!
+```ruby
+source 'https://github.com/CocoaPods/Specs.git'
+platform :ios, '8.0'
+use_frameworks!
 
-  target 'YourApp' do
-      pod 'Sentry', :git => 'https://github.com/getsentry/sentry-cocoa.git', :subspecs => ['Core', 'KSCrash'], :tag => '3.x.x'
-  end
-  ```
+target 'YourApp' do
+    pod 'Sentry', :git => 'https://github.com/getsentry/sentry-cocoa.git', :subspecs => ['Core', 'KSCrash'], :tag => '3.x.x'
```

Comment:
Surprised that prettier kept 4 spaces indentation here, Ruby code typically uses 2 spaces.

Reply:
Yeah, huh. Dunno. Will fix.